capstone/src/main/java/com/google/sps/servlets/UpdateSeatsServlet.java

```diff
@@ -0,0 +1,62 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import com.google.sps.data.Driver;
+import com.google.sps.data.Rider;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.json.JSONString; 
+
+//servlet responsible for updating seats once a rider adds themselves to a car
+@WebServlet("/update-seats")
+public class UpdateSeatsServlet extends HttpServlet { 
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Long driverId = Long.parseLong(request.getParameter("driverId"));
+    Query query = new Query("Driver");
```

Comment:
There's probably some way to phrase this so that you pass in the id you're looking for and the query returns only that driver (or nothing if it's not there).

Reply:
Hmm I will look into this!